packages/server/src/cloud/aws/metadata.ts

```diff
@@ -0,0 +1,32 @@
+import { randomUUID } from 'crypto';
+import { globalLogger } from '../../logger';
+
+let instanceId: string | undefined = undefined;
+
+export async function fetchInstanceId(): Promise<string> {
```

Comment:
Can we use hostname rather than this env var?  I believe the hostname will have largely have the same value (maybe not the exact same string, but something that could be correlated to ECS instance), and that would then be more general purpose for other cloud providers.

Starting to think more about this as we have some contractual obligations for GCP and Azure on the horizon.